### PR TITLE
WIP: Clean up math a bit

### PR DESCRIPTION
**Pull request by BenjaminPrice**
Local Gas Usage: 397109 -> 396686 (=423)

That first commit should have read `remove the min function`.

### CODE REVIEW

File: contracts/LoopringProtocolImpl.sol [hunk 1]
```diff
@@ -881,12 +881,12 @@ contract LoopringProtocolImpl is LoopringProtocol {
         returns (uint)
     {
         var token = ERC20(tokenAddress);
-        return token.allowance(
+        var allowance = token.allowance(
```
Comment (dong77) [inline contracts/LoopringProtocolImpl.sol]: change var to uint?

File: contracts/lib/MathUint.sol [hunk 2]
```diff
@@ -85,14 +81,14 @@ library MathUint {
         }
 
         uint cvs = 0;
+        uint s = 0;
         for (i = 0; i < len; i++) {
-            uint sub = 0;
             if (arr[i] > avg) {
-                sub = arr[i] - avg;
+                s = arr[i] - avg;
```
Comment (dong77) [inline contracts/lib/MathUint.sol]: how about `s = arr[i] > avg ? arr[i] - avg : avg - arr[i];`

File: contracts/LoopringProtocolImpl.sol [hunk 3]
```diff
@@ -866,7 +866,7 @@ contract LoopringProtocolImpl is LoopringProtocol {
             require(order.amountS > 0); // "amountS is zero");
             require(order.amountB > 0); // "amountB is zero");
 
-            state.fillAmountS = order.amountS.min(state.availableAmountS);
+            state.fillAmountS = (order.amountS < state.availableAmountS ? order.amountS : state.availableAmountS);
```
Comment (dong77) [inline contracts/LoopringProtocolImpl.sol]: please try to wrap at or before 80 chars.

File: contracts/LoopringProtocolImpl.sol (hunk 1 above)
Comment (BenjaminPrice) [reply contracts/LoopringProtocolImpl.sol]: Sure, I'll change that. Compiler should handle that though.

File: contracts/lib/MathUint.sol (hunk 2 above)
Comment (BenjaminPrice) [reply contracts/lib/MathUint.sol]: I'll see if it's any different

**Comment by BenjaminPrice** [comment]
Requested changes made

**Pull request merged**